crds/operators.coreos.com_catalogsources.yaml

```diff
@@ -90,6 +91,11 @@ spec:
                   a registry-server with. Only used when SourceType = SourceTypeGrpc.
                   If present, the address field is ignored.
                 type: string
+              priority:
+                description: Priority represents weighting catalogs for the dependency
+                  resolver
+                type: integer
+                format: int32
```

Comment:
Should we use int32? Is int ok? is there other fields also using int 32? Wdyt?

Reply:
We could stick to `int`. [Similar `weight` usage ](https://github.com/kubernetes/community/blob/b3349d5b1354df814b67bbdee6890477f3c250cb/contributors/design-proposals/scheduling/podaffinity.md#api)

Reply:
The important thing here is that the range of acceptable values is precisely defined. If this field has type `integer`, and there's no additional information about which integers are acceptable, then the API is ambiguous.

Reply:
i'm ok either way, just want to compare to what is currently done.

Reply:
Actually, according to https://book.kubebuilder.io/reference/markers/crd-validation.html, it seems like we can only us int for validation reasons?

Reply:
In the reference Harish found, it's not ambiguous because the API states the range of allowable integers precisely.

Reply:
right, right, but jumped thread here, i'm more saying that we should use `int`

Reply:
The range would be bounded using [CRD validation markers](https://book.kubebuilder.io/reference/markers/crd-validation.html)